### PR TITLE
docs: add tool use for Command concepts

### PR DESCRIPTION
**Pull request by vbarda**
To be merged after #2656

### CODE REVIEW

File: docs/docs/concepts/low_level.md [hunk 1]
```diff
@@ -339,50 +339,44 @@ def my_node(state: State) -> Command[Literal["my_other_node"]]:
...
 
-def my_node(state: State) -> Command[Literal["my_other_node"]]:
-    return Command(update={"foo": "bar"}, goto="my_other_node")
+    When returning `Command` in your node functions, you must add return type annotations with the list of node names the node is routing to, e.g. `Command[Literal["my_other_node"]]`. This is necessary for the graph compilation and rendering, and tells LangGraph that `my_node` can navigate to `my_other_node`.
+
+Check out this [how-to guide](../how-tos/command.ipynb) for an end-to-end example of how to use `Command`.
 
-def my_other_node(state: State):
-    return {"foo": state["foo"] + "baz"}
+### Human-in-the-loop
```
Comment (eyurtsev) [inline docs/docs/concepts/low_level.md:356]: Let's remove the HIL until the HIL is ready? -- you could fork the PR so i have a reference?

File: docs/docs/concepts/low_level.md [hunk 2]
```diff
@@ -339,50 +339,44 @@ def my_node(state: State) -> Command[Literal["my_other_node"]]:
...
 
-With `Command` you can also achieve dynamic control flow behavior (identical to [conditional edges](#conditional-edges)):
+A common use case is updating graph state from inside a tool. For example, in a customer support application you might want to look up customer account number or ID in the beginning of the conversation. To update the graph state from the tool, you can return `Command(update={"my_custom_key": "foo", "messages": [...]})` from the tool:
 
 ```python
-def my_node(state: State) -> Command[Literal["my_other_node"]]:
-    if state["foo"] == "bar":
-        return Command(update={"foo": "baz"}, goto="my_other_node")
+@tool
+def lookup_customer_id(customer_name: str):
```
Comment (eyurtsev) [inline docs/docs/concepts/low_level.md:366]: The look up is USUALLY inverted (from customer id we go to customer name)

File: docs/docs/concepts/low_level.md (hunk 1 above)
Comment (vbarda) [reply docs/docs/concepts/low_level.md:356]: yea i think it's ok to remove! will just send you this passage

File: docs/docs/concepts/low_level.md (hunk 2 above)
Comment (vbarda) [reply docs/docs/concepts/low_level.md:366]: yup can do that

File: docs/docs/concepts/low_level.md [hunk 3]
```diff
@@ -353,6 +353,30 @@ def my_node(state: State) -> Command[Literal["my_other_node"]]:
 
 Check out this [how-to guide](../how-tos/command.ipynb) for an end-to-end example of how to use `Command`.
 
+### Using inside the tools
```
Comment (jacoblee93) [inline docs/docs/concepts/low_level.md:356]: Just `Using inside tools`

Comment (vbarda) [inline docs/docs/concepts/low_level.md:356]:
```suggestion
### Using inside tools
```

**Pull request merged**